site/docs/tasks/control-plane-security.md

```diff
@@ -112,6 +115,9 @@ Here is an example of the tls related fields in the opa-connector section that a
 ```bash
 opaConnector:
   tls:
+    # MinVersion contains the minimum TLS version that is acceptable.
+    # If not provided, minimal supported TLS protocol is taken as the minimum.
```

Comment:
what does it mean "If not provided, minimal supported TLS protocol is taken as the minimum." ?
for golang the defaults are:

> By default in goLang, TLS 1.2 is currently used as the minimum when acting as a client, and TLS 1.0 when acting as a server. TLS 1.0 is the minimum supported by this package, both as a client and as a server.

Reply:
maybe add the default minimum value to the requirements.

Reply:
I changed it to use the system default value if it is not defined.